doc/usage/restructuredtext/field-lists.rst

```diff
@@ -9,19 +9,38 @@ fields marked up like this::
 
     :fieldname: Field content
 
-Sphinx provides custom behavior for bibliographic fields compared to docutils.
+Field lists are :duref:`originally defined in docutils <bibliographic-fields>`
```

Comment:
Incorrect. Field-lists do not have a special responsibility to show information about the page. They are only one of document structures like lists, tables, images, and so on.

I think this sentence should mention to the bibliographic fields instead. The bibliographic fields (a.k.a. docinfo) is a special field list that is placed at the top of the document. And they have a special responsibility to describe the page.

Reply:
Okay, yeah, this is confusing now. Good spot. Previously, we talked about field lists in general and then went on to talk about the special "file-wide metadata" behaviour that kicks in when they are at the top of the file. Now we're muddling them up. Could we keep this section before "File-wide metadata" mostly as-is, just adding the note about field lists being parsed as strings? The rework you've done of the next section makes sense, IMO

Reply:
+1 for keeping this section as-is.

Reply:
OK - will remove the first sentence and fix this